tests/unit/src/main/scala/tests/codeactions/BaseCodeActionLspSuite.scala

```diff
@@ -54,19 +56,31 @@ abstract class BaseCodeActionLspSuite(
       changeFile: String => String = identity,
       expectError: Boolean = false,
       filterAction: CodeAction => Boolean = _ => true,
+      scalaCli: Boolean = false,
```

Comment:
Could you maybe add layout as a parameter? Boolean `scalaCLi` is not clear on what it does.

Reply:
In `BaseCodeActionLspSuite` layout is created base on few parameters such as: `scalacOptionsJson`, `scalafixConf`, `path`, `input`. I'm not sure what you expect, do you want that users how to pass explicitly `layout: String`?

For now, I changed name from `scalaCli` to `scalaCliLayout`.